Classes/Domain/Repository/AbstractRepository.php

```diff
@@ -33,4 +33,27 @@ public function crossClientFindAll($returnRawQueryResult = TRUE) {
         $query = $this->createQuery();
         return $query->execute($returnRawQueryResult);
     }
-}
+
+    /**
+     * Finds all records with embargo date
+     * @return mixed
+     */
+    public function crossClientEmbargoFindAll() {
```

Comment:
Placing this method here implies that there can be embargo on just about any object that is handled by any kind repository. Is that correct?

Reply:
I placed this method in the AbstractRepository because there was already a cross client method.
But you are right. Embargo is document only and this method should be placed in the documentRepo.